src/webview/wkwebview/download.rs

```diff
@@ -102,6 +102,7 @@ pub extern "C" fn download_did_finish(this: &Object, _: Sel, download: id) {
 pub extern "C" fn download_did_fail(this: &Object, _: Sel, download: id, error: id, _: id) {
   unsafe {
     let description: id = msg_send![error, localizedDescription];
+    #[allow(unused_variables)]
```

Comment:
How about we move these to `debug_assertions` below? And rename `error` to `_error`.

```
#[cfg(debug_assertions)]
    {
        let description: id = msg_send![_error, localizedDescription];
        let description = NSString(description).to_str().to_string();
        eprintln!("Download failed with error: {}", description);
    }
```

Reply:
@wusyong I revised my patch and force-pushed as e73c4e6